guides/V1_space_template_main.md

```diff
@@ -0,0 +1,21 @@
+---
+copyright: 'Copyright IBM Corp. 2018'
+link: 'space-template-main'
+is: 'experimental'
+---
+
+# Space Template
+
+In Watson Work Services a space template is a template for the creation of a space.
+The space template defines user configurable properties for the space, 
+as well as a set of required apps that are automatically added to the space on creation.
+
+# Key fields
+
+| property      | type          | description  |
+| ------------- |:------------- |:-----|
+| properties |SpacePropertyCollection |There are three types of properties in a space template: List, Boolean, and Text.|
```

Comment:
> SpacePropertyCollection

Is this structure `SpaceProperty` going to be described elsewhere?  If not, it should be part of this doc...

Reply:
I wasn't planning to, as much as possible I would like to rely on the self-documention of the schema. I don't think it adds anything to duplicate here. Maybe it's better not to even mention the type here?

Reply:
It is ok to mention types.... taking the other comment with regards to the API explorer.... Not sure what @jarusso suggests here, but I think we should then include a general pattern/practice of making a reference to the API explorer (with link) for further info.  

I agree in not duplicating (and maintaining) duplicate structures if the properties are obvious such as `spaceTitle`.  Sometimes they are not as obvious either in meaning or how they are intended to be used and thus having a 'guide' to _expand_ rather than _duplicate_ what is on the API explorer may be a good idea.

Reply:
We can add a description of the subtypes à la https://github.com/watsonwork/watsonwork-developer-docs/blob/master/guides/V1_search_main.md

Do you have a list of fields that are really unclear as a guide for us, I might be too familiar with the types.